src/app/ReadTideIndex.cpp

```diff
@@ -52,7 +52,10 @@ int ReadTideIndex::main(int argc, char** argv) {
   }
 
   const pb::Header::PeptidesHeader& pepHeader = peptides_header.peptides_header();
-  MassConstants::Init(&pepHeader.mods(), &pepHeader.nterm_mods(), &pepHeader.cterm_mods(),
+  //TODO: replace nullptr with actual tables.
```

Comment:
Comment by bmx8177: In src/app/ReadTideIndex.cpp at line you have a comment at line 55 about removing nullptrs. I don't see those nullptrs. Is this a stray comment?

Reply:
Yes, I replaced the null pointers, but did not remove the comment. It should be removed.